docs/bigquery-cdcTarget.md

```diff
@@ -16,3 +22,24 @@ The service account key can be found on the Dashboard in the Cloud Platform Cons
 Make sure the account key has permission to access BigQuery.
 The service account key file needs to be available on every node in your cluster and
 must be readable by all users running the job.
+
+Properties
+----------
+
+**Project**: Project of the BigQuery dataset. When running on a Dataproc cluster, this can be left blank,
+which will use the project of the cluster.
+
+**Staging Bucket**: GCS bucket to write change events to before loading them into staging tables.
+This bucket can be shared across multiple replicators. The bucket must be in the same location as the
+BigQuery dataset.
+
+**Service Account Key**: The contents of the service account key to use when interacting with GCS and
+BigQuery. When running on a Dataproc cluster, this can be left blank, which will use the service account
+of the cluster.
+
+**Max Batch Seconds**: Maximum number of seconds to wait before merging a batch of changes.
```

Comment:
Should we also mention default number here?

Reply:
we no longer mention defaults in the docs, and instead set defaults directly in the widget.